packages/frontend/src/components/VisualizationConfigs/common/Config.tsx

```diff
@@ -20,6 +21,12 @@ const Heading: FC<PropsWithChildren> = ({ children }) => (
     </Text>
 );
 
+const Subheading: FC<PropsWithChildren> = ({ children }) => (
```

Comment:
I had this a long time ago but removed because wasn't using it - finally found a place where this is necessary!

Reply:
🙌